docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/gitea/index.md

```diff
@@ -0,0 +1,130 @@
+---
+sidebar_label: Gitea
+---
+
+# The Gitea Webhook Receiver
+
+The Gitea Webhook Receiver will respond to `push` events originating from
+Gitea repositories.
+
+In response to a `push` event, the receiver "refreshes" Warehouses subscribed to
+the Git repository from which the event originated.
+
+:::info
+"Refreshing" a `Warehouse` resource means enqueuing it for immediate
+reconciliation by the Kargo controller, which will execute the discovery of
+new artifacts from all repositories to which that `Warehouse` subscribes.
+:::
+
```

Comment:
For some of the other receivers, if there are multiple hosting options, we mention in an infobox around here compatibility with those options (assuming it is indeed compatible with them all). For Gitea, I know there's cloud and Enterprise. But Gitea's also OSS... I think self-hosted non-Enterprise is also a thing.